packages/flow-core/src/components/f-button/f-button.scss

```diff
@@ -165,19 +165,41 @@ in this case it is `f-button`
 					}
 				}
 			}
+
+			&[state="#{$state}"][category="packed"] {
```

Comment:
you can do to avoid code duplication
&[state="#{$state}"][category="transparent"],&[state="#{$state}"][category="packed"]